aqt/main.py

```diff
@@ -1104,10 +1104,14 @@ def onCheckMediaDB(self):
         if warnings:
             report += "\n".join(warnings) + "\n"
         if unused:
+            numberOfUnusedFilesLabel = str(len(unused))
             if report:
                 report += "\n\n\n"
-            report += _(
-                "In media folder but not used by any cards:")
+            report += numberOfUnusedFilesLabel + " "
+            report += ngettext(
+                "file found in media folder not used by any cards:",
+                "files found in media folder not used by any cards:",
+                int(numberOfUnusedFilesLabel))
```

Comment:
The number needs to be inside the ngettext() strings with %d instead of adding it to the resulting string, as some languages won't have the number at the start of the string.

Reply:
Ah I hadn't considered that, it should be fixed and the ngettext() call should look a little better now.